dask/dataframe/core.py

```diff
@@ -2713,6 +2713,11 @@ def groupby(self, by=None, **kwargs):
     def count(self, split_every=False):
         return super(Series, self).count(split_every=split_every)
 
+    @derived_from(pd.Series)
+    def explode(self):
+        meta = self._meta_nonempty.explode()
```

Comment:
If a user is running with pd.version < 0.25 this will throw an AttributeError:

> AttributeError: 'Series' object has no attribute 'explode'

You might consider catching the error and raising with a nicer message instructing the user to update pandas to use this feature

Reply:
Would you also mind trying to use `self._meta` instead of `self._meta_nonempty`?  This should work and `_meta` is generally preferred

Reply:
Fixed. The error message will show like `NotImplementedError: Base package doesn't support 'explode'. Use pandas 0.25.0 or later to use this method.`

Reply:
If you wouldn't mind me asking what's the difference between `_meta` and `_meta_nonempty`?

Reply:
`_meta` is an empty dataframe with the right types. `_meta_nonempty` is a dataframe with the right types, but a couple rows of dummy data. We use `_meta_nonempty` to work around cases where pandas has different behavior for empty vs. non-empty dataframes. But we prefer to use `_meta` where possible.